specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json

```diff
@@ -423,7 +423,7 @@
             "values": [
               {
                 "value": "RespectQuota",
-                "description": "Ingestion enabled following daily cap quota reset, or subscription enablement."
+                "description": "Renewed ingestion following daily cap quota reset, or subscription activation."
```

Comment:
It's not renewed, it's the stable state where workspace has not ingested more than the quota assigned.
"Daily quota has not been reached."

Reply:
In this case:
"Renewed data ingestion following daily cap quota reset, or subscription enablement."

Reply:
I don't think we need to specify what could have happened here. This status specifies a steady state where data is ingested and quota has not been exceeded. Why complicate with renew, daily cap reset, subscription enablement and all those fancy words.